app/templates/eventView.html

```diff
@@ -29,82 +29,94 @@ <h1 id="title">{{page_title}}</h1>
           </ul>
          </div>
     {% endif %}
-    {% if isPastEvent %}
-        <p class="heading-size"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</p>
-    {% else %}
-        <p class="heading-size"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</p>
-    {% endif %}
-            <div>
-                <div>
-                    <h3><br>Description:</h3>{{eventData.description}}<br><br>
-                    <div>
+            <div class="row">
+                    <div style="padding-right:150px;"class="col-md-6">
```

Comment:
The padding-right here gets a bit noticeable on smaller screens and especially mobile. Taking out the padding looked okay. If you still want padding, I'd suggest using rem or similar, not pixels.